app/assets/javascripts/select-kit/addon/components/filter-for-more.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if this.collection.content.shouldShowMoreTip}}
+  <div class="filter-for-more">
+    {{i18n "select_kit.components.filter_for_more"}}
+  </div>
+{{/if}}
```

Comment:
Can we make this into a glimmer/gjs component?

```suggestion
import i18n from "discourse-common/helpers/i18n";

const FilterForMore = <template>
  {{#if @collection.content.shouldShowMoreTip}}
    <div class="filter-for-more">
      {{i18n "select_kit.components.filter_for_more"}}
    </div>
  {{/if}}
</template>

export default FilterForMore;
```

Reply:
Yes, done!